roles/testnode/tasks/pip.yml

```diff
@@ -1,16 +1,18 @@
 ---
+# python-pip is installed from epel repo which doesn't exist on fedora.
+# package is installed during kickstart instead.
```

Comment:
Do you mean "package is part of default installation on Fedora" here?  (that is, please change it to something like that)